content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -9,46 +9,39 @@ Configure the feature
 =====================
 
 To configure the feature go to :menuselection:`Helpdesk --> Settings --> Helpdesk Teams --> Edit`
-and enable *Ticket closing*.
+and enable :guilabel:`Ticket closing`.
 
-.. image:: close_tickets/closetickets1.png
-   :align: center
+.. image:: close_tickets/close-ticket-settings.png
    :alt: Ticket closing in Odoo Helpdesk
 
 In order to designate to which stage the ticket migrates to once it is closed, go to
 :menuselection:`Helpdesk --> Overview --> Tickets`.
 
-.. image:: close_tickets/closetickets2.png
-   :align: center
+.. image:: close_tickets/customer-care-tickets.png
    :alt: Ticket closing in Odoo Helpdesk
 
 You can either create a new Kanban stage or work with an existing one. For both scenarios, go to
-:menuselection:`Helpdesk --> Settings --> Edit Stage` and enable *Closing Stage*.
+:menuselection:`Helpdesk --> Settings --> Edit Stage` and enable :guilabel:`Closing Stage`.
 
-.. image:: close_tickets/closetickets3.png
-   :align: center
+.. image:: close_tickets/edit-column-closing-stage.png
    :alt: Ticket closing in Odoo Helpdesk
 
-If a closing stage is not specified, by default, the ticket is moved to the last stage;
-contrarily, if you have more than one stage set as closing, the ticket is put in the first one.
+If a closing stage is not specified, by default, the ticket is moved to the last stage; contrarily,
+if you have more than one stage set as closing, the ticket is put in the first one.
 
-The Costumer Portal
+The customer portal
 ===================
 
-Now, once the user logs into his Portal, the option *Close this ticket* is available.
+Now, once the user logs into their portal, the option :guilabel:`Close this ticket` is available.
 
-.. image:: close_tickets/closetickets4.png
-   :align: center
+.. image:: close_tickets/customer-view-close-ticket.png
    :alt: Ticket closing in Odoo Helpdesk
```

Comment:
I would retake this screenshot. It should be zoomed in more and resized. The "Close this ticket" button is supposed to be the focus of this image, but it's so small that you can barely read it.

Reply:
Remember to keep the image width 768px or below; you can play with the browser window width to try to get the correct sizing. For zoom, keep it in the 110% to 125%.